x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -331,80 +336,105 @@ export class Explorer extends React.Component {
               setSelectedCells={this.props.setSelectedCells}
             />
             <EuiSpacer size="m" />
-
             {annotationsData.length > 0 && (
               <>
-                <EuiTitle className="panel-title">
-                  <h2>
-                    <FormattedMessage
-                      id="xpack.ml.explorer.annotationsTitle"
-                      defaultMessage="Annotations"
-                    />
-                  </h2>
-                </EuiTitle>
-                <AnnotationsTable
-                  annotations={annotationsData}
-                  drillDown={true}
-                  numberBadge={false}
-                />
-                <AnnotationFlyout />
-                <EuiSpacer size="l" />
+                <EuiPanel>
```

Comment:
In the Anomaly Explorer, the 'edit annotation' flyout is getting rendered inside this panel. Possibly missing the `EuiOverlayMask' ?


![image](https://user-images.githubusercontent.com/7405507/86465565-cedb2a80-bd29-11ea-83d6-9f9e9811cddf.png)

Reply:
I ended up moving it out of the EuiPanel here https://github.com/elastic/kibana/pull/70198/commits/1f2684dbbc91ca4a7d83ab93d8d4cfb79a5180c2 which I think might be clearer since the flyout isn't necessarily coupled with the table and to avoid the overhead cost. But I can update it the other way if needed.

Reply:
The way you've implemented this is good, as in moving it out of the EuiPanel, and increasing it to half page width also looks good.